db/migrate/20181105061336_cast_key_types_and_values_in_parameters.rb

```diff
@@ -32,13 +32,10 @@ def override_key_type_and_value(param)
   def save_param(param, new_value, new_key_type)
     if Parameter::KEY_TYPES.include?(new_key_type)
       param.class.without_auditing do
-        param.value = new_value
-        param.key_type = new_key_type
-        result = param.save(:validate => false)
+        result = param.update_columns(value: new_value, key_type: new_key_type)
         return if result
       end
       say "Failed to update param(#{param.id}): #{param.inspect}"
-      say "Error: #{param.errors.full_messages.inspect}"
```

Comment:
As `update_columns` skips validations as well as callbacks, I think this line  is not needed here.